File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int ans[]=new int[2];

        for(int i=0;i<nums.length-1;i++){
            int sum=0;
            for(int j=i+1;j<nums.length;j++){
                sum=nums[i]+nums[j];
                if(sum==target){
                    ans[0]=i;
                    ans[1]=j;
                    break;
                }
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:

        r=[1]*len(nums)
        l=[1]*len(nums)

        r[0]=1
        l[len(nums)-1]=1

        for i in range(1,len(nums),1):
            r[i]=r[i-1]*nums[i-1]

        for j in range(len(nums)-2,-1,-1):
            l[j]=l[j+1]*nums[j+1]

        for k in range(len(nums)):
            nums[k]=r[k]*l[k]

        print(nums)
        return nums